docs/UserGuide.md

```diff
@@ -3,8 +3,22 @@ layout: page
 title: User Guide
 ---
 
-PeopleSoft is a desktop app for **calculating the salary for shift-based contractors**, optimized for use via a **Command Line Interface (CLI)** while still having the benefits of a **Graphical User Interface (GUI)**. If you can type fast, PeopleSoft can get your payroll tasks done much faster than traditional GUI apps.
+PeopleSoft is a desktop app for **calculating the salary for shift-based contractors**, optimized for use via a **Command Line Interface (CLI)** If you are a **HR manager** and you can type fast, PeopleSoft can get your payroll tasks done **much faster** than traditional GUI apps.
 
+You can key in your employees' data and the jobs that they are working on. Then, you should assign the employees to the respective jobs that they are working on. After the job has been completed, you can mark the job as paid, and PeopleSoft will calculate how much each employee deserves based on their hourly rates. You can also generate a PDF payslip for you to send to the employees.
```

Comment:
"a PDF payslip to send to the employees"?

"to be sent to the employees"?

"for your employees to reference"?